pmoves/services/agentgym-rl-coordinator/app.py

```
@@ -1,68 +1,356 @@
+
+    if not hf_publisher:
+        raise HTTPException(status_code=503, detail="HuggingFace publisher not available")
+
+    try:
+        result = await hf_publisher.publish_to_huggingface(
+            dataset_name=dataset_name,
+            trajectory_ids=trajectory_ids,
+            session_id=session_id,
+            private=private,
+        )
+        return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)